src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Color {
+    red,
+    green,
+    black,
+    grey,
+    brown,
+    violet,
+    white,
+    orange,
+    yellow,
+    pink

```

Comment:
Enum constants should be in uppercase. For example, `RED` instead of `red`. This is because enum constants are constants and should follow the conventions for constants.